src/buildstream/_protos/build/bazel/remote/asset/v1/remote_asset_pb2.py

```diff
@@ -4,9 +4,8 @@
 """Generated protocol buffer code."""
 from google.protobuf import descriptor as _descriptor
 from google.protobuf import descriptor_pool as _descriptor_pool
-from google.protobuf import message as _message
```

Comment:
I think we were using a specific version of protobuf compiler to generate these so that they work with both old and new grpc.

Reply:
Is it documented which version this was?

Reply:
Looks like it was 3.20.1. This is btw *really* dangerous, the grpcio-tools that is compatible with this old protobuf is getting increasingly hard to install. It would be better to drop support for older grpc.

Reply:
Yay. grpcio-tools needed doesn't build anymore with modern g++.